package.json

```diff
@@ -4,10 +4,12 @@
   "description": "A Javascript library for parsing metadata in a Web Page.",
   "main": "parser.js",
   "scripts": {
-    "test:lint": "./node_modules/eslint/bin/eslint.js ./*.js ./tests/*.js",
-    "test:mocha": "./node_modules/istanbul/lib/cli.js cover _mocha --report lcovonly -- tests/* -R spec",
-    "test": "./node_modules/npm-run-all/bin/npm-run-all/index.js test:*",
-    "cover": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
+    "test:lint": "eslint ./*.js ./tests/*.js",
+    "test:karma": "karma start",
+    "test:mocha": "./node_modules/istanbul/lib/cli.js cover _mocha --report lcovonly -- tests/*.test.js -R spec",
```

Comment:
you _may_ be able to replace `./node_modules/instanbul/lib.cli.js` with just `istanbul`.
running `ls -las ./node_modules/.bin/` shows that there is a symlink from `instanbul -> ../instanbul/lib/cli.js`.